connman/src/service.c

```diff
@@ -9794,6 +9853,8 @@ int __connman_service_init(void)
 
 	service_hash = g_hash_table_new_full(g_str_hash, g_str_equal,
 							NULL, service_removed);
+	service_type_hash = g_try_new0(GHashTable*,
```

Comment:
Why bother with try new if the result is never checked?

Reply:
Yep, only `g_new0` is to be used with small allocations, if it fails then it is deemed to fail as per to upstream coding conventions as well. Fixed with 97c3b643978731057463a21799e8bd362fb4a261